.github/workflows/deploy.yml

```
@@ -0,0 +1,50 @@
+name: Deploy `dandidav` to Heroku
+
+on:
+  push:
+    branches:
+      - main
```

```suggestion
  pull_request_target:
    branches:
      - main
    types:
      - closed
```

This is needed for the `if:` below to work properly.